components/analytics/bot-error-analysis.tsx

```
@@ -0,0 +1,750 @@
+                  <div className="flex items-center justify-between">
+                    <h4 className="font-medium">Error Priorities</h4>
+                    <span className="text-xs text-muted-foreground">Count</span>
+                  </div>
+                  <div className="space-y-2 max-h-[400px] overflow-y-auto pr-2">
+                    {errorsByPriority.map((priority) => (
+                      <button
+                        key={priority.name}
+                        className={`w-full flex items-center justify-between p-2 rounded-md hover:bg-secondary/50 transition-colors ${selectedPriority === priority.name ? "bg-secondary" : ""
+                          }`}
+                        onClick={() => setSelectedPriority(priority.name === selectedPriority ? null : priority.name)}
+                      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)